blog/2023-01-09-Apache-Pulsar-2-10-3.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "What’s New in Apache Pulsar 2.10.3"
+date: 2023-01-09
+author: "liangyepianzhou, momo-jun"
+---
+
+The Apache Pulsar community releases version 2.10.3! 50 contributors provided improvements and bug fixes that delivered 155 commits. Thanks for all your contributions.
+
+This blog walks through the most noteworthy changes. For the complete list including all feature enhancements and bug fixes, check out the [Pulsar 2.10.3 Release Notes](https://pulsar.apache.org/release-notes/versioned/pulsar-2.10.3/).
+
+### Add `getState` in transactions for client APIs ([PR-17423](https://github.com/apache/pulsar/pull/17423))
+
+#### Issue
+In earlier versions, the state of transactions on the client side could not be obtained with the client APIs. Users have no way to check the state of transactions before ending them.
```

Comment:
Shall we consider using "past tense" since they were facts/actions that existed/took in the past?

For example, https://pulsar.apache.org/blog/2022/04/08/Apache-Pulsar-2-9-2/

<img width="903" alt="image" src="https://user-images.githubusercontent.com/50226895/212218528-af6efa78-6193-4ca5-8565-0aac1e301f7d.png">

Reply:
Thanks for raising this question. Here is my thought: users who are using the earlier versions still face the issue today. It was not past for those who may need an upgrade.